scripts/code_coverage_report/generate_code_coverage_report/Sources/IncrementalCoverageReportGenerator/main.swift

```diff
@@ -151,42 +156,41 @@ struct IncrementalCoverageReportGenerator: ParsableCommand {
         )
         for addedLineIndex in change.addedLines {
           if addedLineIndex < coverageFile.coverage.count {
-            if let test_cover_run = coverageFile.coverage[addedLineIndex] {
-              if test_cover_run == 0 { uncoveredLine.coverage.append(addedLineIndex) }
-              print("\(addedLineIndex) : \(test_cover_run)")
+            if let testCoverRun = coverageFile.coverage[addedLineIndex] {
+              if testCoverRun == 0 { uncoveredLine.coverage.append(addedLineIndex) }
             }
```

Comment:
This can be combined into a single `if` statement:

```
if let testCoverRun = coverageFile.coverage[addedLineIndex], testCoverRun == 0 {
  uncoveredLine.coverage.append(addedLineIndex)
}
```